src/hotspot/cpu/riscv/sharedRuntime_riscv.cpp

```diff
@@ -1004,17 +1004,20 @@ static void gen_continuation_enter(MacroAssembler* masm,
 
     __ bnez(c_rarg2, call_thaw);
 
-    // Make sure the call is patchable
-    __ align(NativeInstruction::instruction_size);
-
-    const address tr_call = __ reloc_call(resolve);
-    if (tr_call == nullptr) {
-      fatal("CodeCache is full at gen_continuation_enter");
-    }
+    address tr_call;
```

Comment:
Can you rename this to `call_pc` while you are on it? `tr_call` is named after trampoline call, but we don't have that now.